p2p/src/peer_manager/config.rs

```diff
@@ -0,0 +1,130 @@
+// Copyright (c) 2021-2024 RBB S.r.l
+// opensource@mintlayer.org
+// SPDX-License-Identifier: MIT
+// Licensed under the MIT License;
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// https://github.com/mintlayer/mintlayer-core/blob/master/LICENSE
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use std::time::Duration;
+
+use utils::make_config_setting;
+
+use super::{
+    peerdb::config::PeerDbConfig,
+    peers_eviction::{
+        OutboundBlockRelayConnectionMinAge, OutboundFullRelayConnectionMinAge,
+        PreservedInboundCountAddressGroup, PreservedInboundCountNewBlocks,
+        PreservedInboundCountNewTransactions, PreservedInboundCountPing,
+    },
+};
+
+make_config_setting!(MaxInboundConnections, usize, 128);
+make_config_setting!(OutboundFullRelayCount, usize, 8);
+make_config_setting!(OutboundFullRelayExtraCount, usize, 1);
+make_config_setting!(OutboundBlockRelayCount, usize, 2);
+make_config_setting!(OutboundBlockRelayExtraCount, usize, 1);
+make_config_setting!(StaleTipTimeDiff, Duration, Duration::from_secs(30 * 60));
+make_config_setting!(MainLoopTickInterval, Duration, Duration::from_secs(1));
+make_config_setting!(
+    FeelerConnectionsInterval,
+    Duration,
+    Duration::from_secs(2 * 60)
+);
+make_config_setting!(EnableFeelerConnections, bool, true);
+make_config_setting!(ForceDnsQueryIfNoGlobalAddressesKnown, bool, false);
+
+// TODO: this name is too generic, because not all peer manager settings are contained here.
```

Comment:
Let's have a discussion about this at some point when you feel it's appropriate. Using words like "internal" can quickly become a mess.